.npmignore

```diff
@@ -1,3 +0,0 @@
-.git*
-test/
-docs
```

Comment:
Why was .npmignore removed?

Reply:
Yup the recommended method of defining what gets packaged is the `files` array now and thngs like `node_modules` are automatically excluded.